boxoffice/views/custom_exceptions.py

```diff
@@ -1,9 +1,24 @@
 # -*- coding: utf-8 -*-
 
+from .. import mail, app
+from flask.ext.mail import Message
+from flask import jsonify, make_response
+from utils import cors
+
 
 class APIError(Exception):
 
-    def __init__(self, message, status_code):
+    def __init__(self, message, status_code, response_message):
         Exception.__init__(self)
         self.message = message
         self.status_code = status_code
+        self.response_message = response_message
+
+
+@app.errorhandler(APIError)
+@cors
+def handle_api_error(error):
+    msg = Message(subject='API Error', recipients=app.config['ADMINS'])
+    msg.body = error.message
+    mail.send(msg)
```

Comment:
Sure, I'll write to a log file. But, it's also useful for us to know when something like this occurs, right? For which, I thought email works well.